meetings/2023-05-10.md

```diff
@@ -0,0 +1,124 @@
+* Repository: https://github.com/solid/specification
+* Status: Draft
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+* elf Pavlik
+* Jeff Zucker
+* [Hadrian Zbarcea](https://hadrian.solidcommunity.net/profile/card#me)
+---
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/solid/specification/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Sarven
+* Pavlik
+
+### Introductions
+* name: text
+
+---
+
+
+## Topics
+
+### Overview of today's agenda
+
+
+### WIP Implementation Feedback
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+* eP: ...
+* SC: Played with blob and something else IIRC in dokieli's "save as.." for media/binary resources.
+* eP: we use authenticated fetch. We get a blob from response. If you pass that to url createObjectURL, the resulting URL is something you can use in source of the image or as the download file. So far it works. Haven't tried with images that exceed available memory. For now we use that method for most basics and whatever attachments people want to use.
+* eP: In sai-js I use UUID.. no intention to use anything human-readable in the URI. Re download, I plan to have it as part of the resource description - part of the metadata not URI. Possibly also include media type information. CSS takes content-type from the request. It preserves the extension if file-based system used. I was thinking of using content-type in the resource description. If you have some kind of attachment, you could get some kind of icon to represented. one of the motivations no tot URI because to use human-readable label without changing the URI.
+* SC: You mean "Description Resource" right?
```

Comment:
```suggestion
* eP: In `sai-js`, I use UUID... No intention to use anything human-readable in the URI. Re: download, I plan to have it as part of the resource description — part of the metadata, not URI. Possibly also include media type information. CSS takes `content-type` from the request. It preserves the extension, if file-based system used. I was thinking of using `content-type` in the resource description. If you have some kind of attachment, you could get some kind of icon to represent it. One of the motivations not to use URI because to use human-readable label without changing the URI.
* SC: You mean "Description Resource", right?
```